assets/js/feature-tours/dashboard-sharing.js

```diff
@@ -68,7 +68,8 @@ const dashboardSharing = {
 		if (
 			ACTIONS.STOP === action ||
 			ACTIONS.CLOSE === action ||
-			( index === 0 && dialogOpen )
+			( index === 0 && dialogOpen ) ||
+			index + 1 > size
```

Comment:
This doesn't seem to address the issue as `index + 1` would at most equal the `size`.

Reply:
A debugged and there is a case when the index gets 4 on the last step, where we should actually close the modal.
 It should help.